nep29/nep29.py

```diff
@@ -21,6 +21,24 @@
 from packaging.version import Version
 
 
+def get_python_versions_dates():
+    # Python releases for 3.X.0 are hard-coded since they can't be obtained
+    # from PyPI (only concerned with X>=5)
+    format = "%Y-%m-%d"
+    python_releases = {
+        '3.5.0': '2015-09-13',
```

Comment:
Thank you for this, are we able to parse releases on github instead?
https://github.com/python/cpython/tags

It seems that python's official page might be hard to parse.